### DISCUSSION TITLE
C/C++ Console not displaying expected results

### DISCUSSION THREAD

**Discussion by louisCSCD** [eclipse-cdt/cdt#673] [Q&A]
I'm new to the world of C/C++ and Eclipse.  it appears that I've successfully compiled and build the "HelloWorld" application.  However, I'm seeing the results of the application displayed in the console window.  Instead the following message is being displayed: 

openjdk version "17.0.9" 2023-10-17
OpenJDK Runtime Environment Temurin-17.0.9+9 (build 17.0.9+9)
OpenJDK 64-Bit Server VM Temurin-17.0.9+9 (build 17.0.9+9, mixed mode)

I've been investigating the configuration of Preferences and Perspectives and so far have not found a solution.  I would appreciate it  greatly if someone can point it the right direction of a solution.   What I find puzzling is that I'm not doing any work with Java or JDK.  

Thanks

**Comment by foxkicks** [comment]
Are you compiling C/C++ using a Java compiler? You need to use something like GNU or Clang. Otherwise there's not much I can tell you without seeing the code. Eclipse is a Java IDE by default and supports some other languages. Personally, I use Visual Studio.

**Comment by ameyaatmaa** [comment]
I think you should use Vs code or any other compiler as Eclipse is a compiler for mainly Java , you can use it for c/c++ but from my personal experience VS code is much better and handy for both new and advanced developers.
